### DISCUSSION TITLE
error in setting up environment

### DISCUSSION THREAD

**Discussion by NishchayKQ** [efroemling/ballistica#531] [Q&A]
using the guide on - https://github.com/efroemling/ballistica/wiki/Getting-Started
Preparing Your Environment --> Android

#note
i am using latest termux from f-droid as playstore one is no longer recommended 
learn more about this here - https://www.reddit.com/r/termux/comments/pkujfa/important_deprecation_notice_for_google_play/

#error
when i run 
![Screenshot_20221016-163330_Termux](https://user-images.githubusercontent.com/108711354/196031748-4281d431-9974-4e5c-91fc-973f73a8e27b.png)
> apt-get install software-properties-common
it raises error :
```
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package software-properties-common
```

**Comment by Gengar-lab** [comment]
Is your Android Device rooted?

**Reply by NishchayKQ** [reply]
no

**Reply by Gengar-lab** [reply]
Try using the method of Non rooted phones.

**Reply by NishchayKQ** [reply]
im sorry but did u read what i wrote? those are the steps for non rooted devices...

**Reply by Gengar-lab** [reply]
Try ```pkg install software-properties-common```

**Reply by NishchayKQ** [reply]
thx ! though i gave up on that idea as i found out bombsquad still supports /android/data folder mods, and for prototyping nothing beats offline!
